src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/service/CcdSupplementaryDetailsSearchService.java

```diff
@@ -24,18 +31,26 @@
 public class CcdSupplementaryDetailsSearchService implements SupplementaryDetailsService {
     private static final String CASE_TYPE_ID = "Asylum";
     private static final String APPELLANT_FAMILY_NAME = "appellantFamilyName";
-
     private final CoreCaseDataApi coreCaseDataApi;
     private final SystemTokenGenerator systemTokenGenerator;
     private final AuthTokenGenerator s2sAuthTokenGenerator;
+    private final int maxRecords;
 
+    @Autowired
```

Comment:
@bradsorour Couldn't we do this Autowired on constructor? Any reason not to? I think constructor autowiring is a preferred approach in all our services.

Reply:
Thanks for that @jkumar-moj - I've moved both annotations into the constructor.

Reply:
thanks Brad